codebuild/spec/buildspec_integ_matrix.yml

```diff
@@ -0,0 +1,64 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+version: 0.2
+
+# Doc for batch https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html#build-spec.batch.build-list
+env:
+  variables:
+    # CODEBUILD_ is a reserved namespace.
+    CB_BIN_DIR: "./codebuild/bin"
+
+batch:
+  build-matrix:
+    static:
+      ignore-failure: false
+      env:
+        type: LINUX_CONTAINER
+        privileged-mode: false
+        compute-type: BUILD_GENERAL1_LARGE
+
+    dynamic:
+      env:
+        image:
+          - aws/codebuild/standard:4.0
+          #- aws/codebuild/amazonlinux2-x86_64-standard:2.0
+          #- aws/codebuild/amazonlinux2-aarch64-standard:1.0
```

Comment:
can these lines be deleted?

Reply:
Why were these commented out?

Reply:
We can add them later, this is aspirational as we don't have the build working on AmazonLinux2, yet.